app/src/main/res/drawable/myhome.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="24dp" android:tint="#FFFFFF"
```

Comment:
the title of this file should be more descriptive and use underscores to separate words

Reply:
done